zap/src/main/java/org/parosproxy/paros/model/SiteNode.java

```diff
@@ -293,10 +293,10 @@ public String getCleanNodeName(boolean specialNodesAsRegex) {
                 // Strip the param summary off
                 name = name.substring(0, bracketIndex);
             }
-            int quesIndex = name.indexOf("?");
-            if (quesIndex > 0) {
+            int questIndex = name.indexOf("?");
```

Comment:
This is a local variable, it isn't a big deal, but it's just an easier token to my eyes.

Reply:
Better with `query` than `quest`.